src/devices/WebVRDevice.js

```diff
@@ -308,31 +318,11 @@ export default class WebVRDevice extends XRDevice {
     // If the session has an outputContext for magic window and there's not an
     // active immersive session using the same canvas, make sure the underlying
     // WebGL canvas is sized to match the output canvas.
-    if (session.outputContext && !session.immersive) {
```

Comment:
are these no longer needed with the removal of xrpresentationcontext?

Reply:
Yes, but the comment was outdated. Fixed.